src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Header.razor.cs

```diff
@@ -1,6 +1,21 @@
-﻿namespace Bit.Client.Web.BlazorUI.Playground.Web.Pages.Components
+﻿using Microsoft.AspNetCore.Components;
+
+namespace Bit.Client.Web.BlazorUI.Playground.Web.Pages.Components
 {
     public partial class Header
     {
+        private readonly string demoPath = "components";
+        private readonly string getStartedPath = "get-started";
```

Comment:
there is no need to define these variables, plz move the strings into the razor file.